### ISSUE TITLE
Logs not persistent in any environment

### ISSUE THREAD

**Issue by PCatinean** [Tecnativa/docker-odoo-base#121]
All logs are printed to stdout and there is no persistent volume to accommodate them by default, even in an production environment.

This means on every update the logs are lost on container recreation if not backed up specifically.

I've started a local build to save logs to a new persistent volume and also redirect them to stdout. Just wanted to make sure you are interested in such a fix or leaving logs out of a persistent volume was a design choice.

Though to be honest at least the production environment should enforce persistence of logs imo :P 

Let me know if you are interested in me opening a PR for this.

**Comment by pedrobaeza** [comment]
Are you sure they are not persistent with `docker-compose logs ...`?

**Comment by PCatinean** [comment]
From what I've noticed they are kept on the host at /var/lib/docker/containers/<container id>/<container id>-json.log

However when doing an update with a new image and recreating the container the logs are removed along with the container.

Wow that reply was fast :D

**Comment by pedrobaeza** [comment]
Then that's something @Yajo should confirm. If that's true, indeed it can be a problem.

**Comment by Yajo** [comment]
Sorry @PCatinean, we don't care about logs because anything put into STDOUT/STDERR is captured by Docker and sent to [its configured logging driver](https://docs.docker.com/config/containers/logging/configure/). I can see for your comments that [you use the default json logger](https://docs.docker.com/engine/admin/logging/json-file/). I don't recommend using that one for production (actually, nowhere).

So, basically, this depends on your system configuration, and it's not related to the project itself.

Thanks!

**Comment by PCatinean** [comment]
Okay perfect, thanks for the update @Yajo
